lib/diagnostic.rb

```diff
@@ -108,7 +108,9 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = '@Person = Hash.new {favorite_number: 'favorite_number', first_name: 'first_name'}
```

Comment:
This would be with the constructor, we wanted the short hand syntax like this:

`'person = {first_name: "Lauren", favorite_number: 7}'`